ipyvolume/embed.py

```diff
@@ -140,63 +57,118 @@ def save_js(dirname, makedirs=True):
     src = os.path.join(dir_name_src, "index.js")
     shutil.copy(src, dst)
```

Comment:
done, added devmode option to switch on fetching from js/dist/index.js so that, when I can turn this off when running tests